SMART Health Cards.ipynb

```diff
@@ -520,23 +508,48 @@
     }
    ],
    "source": [
+    "// Create array of chunks to convert into numeric mode QRs\n",
     "var QRCode = require('qrcode')\n",
-    "\n",
-    "const segments = [\n",
+    "const jwsChunks = splitJwsIntoChunks(jws);\n",
+    "\n",
+    "// Save QR code data for future comparison with scanned QR Code\n",
+    "let qrCodesData = [];\n",
+    "\n",
+    "jwsChunks.forEach(async function(jwsChunk, i) {\n",
+    "    // This code snippet was adapted/taken from https://github.com/smart-on-fhir/health-cards/blob/152a4f83b223b5fd14027f765e957e290649f2c0/generate-examples/src/index.ts#L173\n",
+    "    const numericJWS = await jws.split('')\n",
+    "        .map((c) => c.charCodeAt(0) - 45)\n",
+    "        .flatMap((c) => [Math.floor(c / 10), c % 10]) //Need to maintain leading zeros\n",
+    "        .join('')\n",
+    "\n",
+    "    const qrCodeData = 'shc:/' + numericJWS\n",
+    "    \n",
+    "    qrCodesData.push(qrCodeData);\n",
+    "    \n",
+    "    const segments = [\n",
     "    {data: 'shc:/', mode: 'byte'},\n",
     "    {data: numericJWS, mode: 'numeric'}\n",
-    "]\n",
-    "\n",
-    "let qrSVG\n",
-    "QRCode.toString(segments,{type: 'svg', errorCorrectionLevel: 'low', version: 22 }).then(function(result){\n",
-    "    qrSVG = result\n",
-    "    $$.svg(qrSVG)\n",
-    "})\n",
+    "    ]\n",
     "\n",
-    "// Also print as PNG for scanning later on.\n",
+    "    let qrSVG\n",
+    "    // QR code version will default to smallest possible version\n",
+    "    await QRCode.toString(segments,{type: 'svg', errorCorrectionLevel: 'low'}).then(function(result){\n",
+    "        qrSVG = result\n",
+    "        $$.svg(qrSVG)\n",
+    "    })\n",
     "\n",
+    "    // Also print as PNG for scanning later on.\n",
     "\n",
-    "QRCode.toFile('./js_qrcode.png', segments, {width: 800, version: 22})"
+    "    await QRCode.toFile(`./js_qrcode${i}.png`, segments, {width: 800})\n",
+    "});"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "undefined-telephone",
+   "metadata": {},
+   "source": [
+    "The above shows what the value of the QR code will be, with the leading `byte` segment and the `numeric` segment following"
```

Comment:
`console.log(qrCodesData)` was removed so we no longer print the value. Can we add the console.log back in?